src/braket/aws/aws_device.py

```diff
@@ -578,16 +579,18 @@ def get_devices(
             aws_session (Optional[AwsSession]): An AWS session object.
                 Default is `None`.
 
+        Raises:
+            ValueError: order_by not in ['arn', 'name', 'type', 'provider_name', 'status']
+
         Returns:
             list[AwsDevice]: list of AWS devices
         """
-
         if order_by not in AwsDevice._GET_DEVICES_ORDER_BY_KEYS:
             raise ValueError(
                 f"order_by '{order_by}' must be in {AwsDevice._GET_DEVICES_ORDER_BY_KEYS}"
             )
         types = (
-            frozenset(types) if types else frozenset({device_type for device_type in AwsDeviceType})
+            frozenset(types) if types else frozenset({device_type for device_type in AwsDeviceType})  # noqa C416
```

Comment:
This is a valid suggestion, we should simplify to `frozenset(AwsDeviceType)`. I've verified this works as expected